### ISSUE TITLE
crashes when going from name strap to one-line text strap

### ISSUE THREAD

**Issue by valbuilds** [playsamay4/Viz2.0#1]
*originally reported by [daniellp021 on Discord](https://discord.com/channels/1158623262922772492/1164588602693537862/1164588602693537862).*

viz 1.1.0 crashes when transitioning from name strap to single-line text strap

**Comment by valbuilds** [comment]
forgot to leave an error mb
```
Error

LowerThirdFull.lua:710: attempt to index local 'Subtitles' (a nil value)


Traceback

[love "callbacks.lua"]:228: in function 'handler'
LowerThirdFull.lua:710: in function 'ShowLowerThirdText'
LowerThirdFull.lua:940: in function 'after'
lib/timer.lua:44: in function 'updateTimerHandle'
lib/timer.lua:63: in function 'update'
main.lua:94: in function 'update'
[love "callbacks.lua"]:162: in function <[love "callbacks.lua"]:144>
[C]: in function 'xpcall'
```

**Comment by playsamay4** [comment]
Should be fixed with fc978454398e8e81bbd74598485e3f9a559b354d :)
